sdk/core/azure-core/tests/perf_tests/_test_base.py

```diff
@@ -0,0 +1,293 @@
+                async_policies.append(policy(sdk_moniker=self.sdk_moniker))
+            async_pipeline = AsyncPipeline(
+                transport=self.async_transport(),
+                policies=async_policies
+            )
+        return AsyncPipelineClient(
+            self.account_endpoint, pipeline=async_pipeline
+        )
+
+    def _set_auth_policies(self):
+        if not self.args.aad:
+            # if tables, create table credential policy, else blob policy
+            if "tables" in self.sdk_moniker:
+                self.sync_auth_policy = TableSharedKeyCredentialPolicy(
+                    AzureNamedKeyCredential(self.account_name, self.account_key)
+                )
+                self.async_auth_policy = self.sync_auth_policy
+            else:
+                self.sync_auth_policy = BlobSharedKeyCredentialPolicy(self.account_name, self.account_key)
+                self.async_auth_policy = self.sync_auth_policy
+        else:
+            sync_credential = ClientSecretCredential(
+                self.tenant_id,
+                self.client_id,
+                self.client_secret
+            )
+            self.sync_auth_policy = BearerTokenCredentialPolicy(
+                sync_credential,
+                self.storage_scope
+            )
+            async_credential = AsyncClientSecretCredential(
+                self.tenant_id,
+                self.client_id,
+                self.client_secret
+            )
+            self.async_auth_policy = AsyncBearerTokenCredentialPolicy(
+                async_credential,
+                self.storage_scope
+            )
+
+    @staticmethod
+    def add_arguments(parser):
+        super(_ServiceTest, _ServiceTest).add_arguments(parser)
+        parser.add_argument(
+            '--transport',
+            nargs='?',
+            type=str,
+            help="""Underlying HttpTransport type. Defaults to `aiohttp` if async, `requests` if sync. Other possible values for async:\n"""
+                 """ - `requests`\n""",
+            default=None
+        )
+        parser.add_argument('-s', '--size', nargs='?', type=int, help='Size of data to transfer.  Default is 10240.', default=10240)
+        parser.add_argument('--policies',
+                            nargs='?',
+                            type=str,
+                            help="""List of policies to pass in to the pipeline. Options:"""
+                            """\n- None: No extra policies passed in, except for authentication policy. This is the default."""
+                            """\n- 'all': All policies added automatically by autorest."""
+                            """\n- "['policy1','policy2',...]": String list of string policies, such as "['RetryPolicy', 'HttpLoggingPolicy'].""",
+                            default=None)
+        parser.add_argument('--content-decode-policy-kwargs',
```

Comment:
Is this still needed? I didn't see it used...

Reply:
nope! removing